brightness-mode-explainer.md

```diff
@@ -121,6 +115,14 @@ Some form of "scannable element" property. When an element with said property is
 <body style="brightness: max">*
 ```
 
+## Security considerations
+
+- The API is available to secure browsing contexts.
+
+- The screen brightness can be controlled only in response to a user gesture to ensure user retain control over their screen brightness. This prevents a situation where a site increases screen brightness every time the system or user overrides it manually.
+
+- If the page visibility becomes hidden after screen brightness has been increased, the screen brightness should be restored automatically.
```

Comment:
I would also add that if the brightness change request is denied, it could/should be denied with a generic reason where it isn't possible to necessarily distinguish why the request failed.

Reply:
Thank you @willmorgan I've actually went further. See my latest patch. What do you think?

Reply:
It would be helpful to know whether the request succeeded or not, but not necessarily why. So we would blanket deny with a NotAllowedError and not go any more granular than that.

Reply:
What would be the requirements to know whether the request succeeded or not?

Thinking out loud: If the screen brightness is already at its maximum and you call `await screen.requestBrightnessIncrease()` would you expect this to fail even it didn't succeed?

I think having this baked in would help adoption of this feature from a security/privacy perspective.

Reply:
Good question. I think if there's no gesture, battery level is too low, or feature policy (assuming we're going to do this) is misconfigured, I'd expect to see a NotAllowedError.

If the brightness is already at 100% and we do `await screen.requestBrightnessIncrease()` I'd expect the promise to fulfil otherwise it might be possible to infer the brightness of the screen. If we allowed that to happen it would contradict the intention to not offer the ability to read the screen brightness.